packages/shared/components/BalanceSummary.svelte

```diff
@@ -1,31 +1,52 @@
 <script lang="typescript">
-    import { Box, Text } from 'shared/components'
+    import { Text, Tooltip } from 'shared/components'
+    import { Unit } from '@iota/unit-converter'
+    import { formatUnitBestMatch, formatUnitPrecision } from 'shared/lib/units'
 
-    export let balance
-    export let transactions
-    export let accounts
-</script>
+    export let color = 'blue' // TODO: profiles will have different colors
+
+    export let balanceRaw
+    export let balanceFiat
+
+    let balanceBox
+    let parentWidth = 0
+    let parentLeft = 0
+    let parentTop = 0
 
-<Box classes="flex flex-col h-full">
-    <Box classes="flex-grow border-solid border-b border-gray-200">
-        <Text type="p" secondary classes="mb-3 uppercase">Balance</Text>
-        <Box classes="flex mb-2">
-            <Text type="h2" classes="uppercase">{balance.split(' ')[0]}</Text>
-            <Text type="p" secondary classes="ml-1">{balance.split(' ')[1]}</Text>
-        </Box>
-        <Text type="h4" highlighted classes="mb-3 uppercase">45000 USD</Text>
-    </Box>
+    let showTooltip = false
+    let showPreciseBalance = false
+
+    function toggleTooltip() {
+        showTooltip = !showTooltip
+        parentWidth = balanceBox.offsetWidth / 2
```

Comment:
We could maybe move all the parent box calculations to a separate function to avoid duplicate code? Something like
```javascript
function refreshParentBox(){
        parentWidth = balanceBox.offsetWidth / 2
        parentLeft = balanceBox.getBoundingClientRect().left
        parentTop = balanceBox.getBoundingClientRect().top
}
```

Reply:
Happy to do this, thats just me copying code from existing functionality 😄